build/scripts/PullRequestValidation/LinkPullRequestToWorkItem.ps1

```diff
@@ -0,0 +1,52 @@
+using module ..\GitHub\GitHubPullRequest.class.psm1
+using module ..\GitHub\GitHubIssue.class.psm1
+
+param(
+    [Parameter(Mandatory = $true)]
+    [string] $PullRequestNumber,
+    [Parameter(Mandatory = $true)]
+    [string] $Repository
+)
+
+function Update-GitHubPullRequest() {
+    param(
+        [Parameter(Mandatory = $false)]
+        [string] $Repository,

```

Comment:
This parameter shouldn't be needed. Or at least I don't see a case when it's not `$PullRequest.Repository`.